client/src/game/api/events.ts

```diff
@@ -96,6 +96,9 @@ socket.on("Position.Set", (data: { x: number; y: number; zoom: number }) => {
 socket.on("Notes.Set", (notes: Note[]) => {
     for (const note of notes) gameStore.newNote({ note, sync: false });
 });
+socket.on("Markers.Set", (markers: Marker[]) => {
```

Comment:
What you receive here is not in fact a Marker, but a Pick<Marker, 'uuid'> or rather just a string[].  The Marker.name aspect does not come from the server.

That said, I have some remarks on the Marker.name below